aviatrix/resource_aviatrix_firewall_instance.go

```diff
@@ -339,8 +340,8 @@ func resourceAviatrixFirewallInstanceCreate(d *schema.ResourceData, meta interfa
 			return fmt.Errorf("'egress_vpc_id' is only valid for GCP")
 		}
 	} else if cloudType == goaviatrix.GCP {
-		if firewallInstance.ManagementVpc == "" {
-			return fmt.Errorf("'management_vpc_id' is required for GCP")
+		if firewallInstance.ManagementVpc == "" && strings.HasPrefix(firewallInstance.FirewallImage, "Palo Alto Networks") {
+			return fmt.Errorf("'management_vpc_id' is required for GCP with Palo Alto Networks Firewall")
```

Comment:
Is it necessary to add a check for "Required to be empty for GCP Check Point or Fortinet series."?

Reply:
Yes I think that is a good check to have. Added and pushed a commit.